File: sorting/buble.js
function bubleSorting(arr){
    for(let i=0;i<arr.length;i++){
        for(let j=0;j<arr.length;j++){
            if(arr[j] > arr[j+1]){
                let temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp
            }
        }
    }
    return arr
}

console.log(bubleSorting([5,2,7,3,2,7,42,675,8]));

File: sorting/buble.cpp
class Solution {
    public:
        void sortColors(vector<int>& nums) {
            size_t n = nums.size();
            for(int i=0;i<n;i++){
                for(int j=0;j<n-1;j++){
                    if(nums[j] > nums[j+1]){
                        int temp = nums[j];
                        nums[j] = nums[j+1];
                        nums[j+1] = temp;
                    };
                };
            };
        };
    };